iquip/apps/builder.py

```diff
@@ -291,13 +291,46 @@ def value(self) -> Optional[str]:
         return None
 
 
+class _ScanEntry(QWidget):
+    """Entry class for a Scannable object.
+    
+    Attributes:
+        name: The name of the experiment.
+        argInfo: Each key and its value are:
+                default: The Dictionary that describes arguments of a specific scanning type.
```

Comment:
If there are many lines of explantation, we have written down only 2 spaces, like [monitor.py](https://github.com/snu-quiqcl/iquip/blob/develop/iquip/apps/monitor.py#L505).

Reply:
And this is just a question, but will you add three more entry classes for other scan types?
I wonder if all scan types have the same `argInfo` structure.

Reply:
I followed other docsting inside builder.py as below.
```
class _EnumerationEntry(_BaseEntry):
    """Entry class for an enumeration value.
    
    Attributes:
        argInfo: Each key and its value are:
            choices: The pre-defined candidates.
            (optional) default: The string value.
              If not exist, the comboBox is set to the first candidate.
        comboBox: The combobox showing the enumeration value.
    """
```
I will change docsting inside ScanEntry code. Do you want to change all the docsting inside builder app?

Reply:
I will add dummy 4 entry classes (Range, Center, NoScan, Explicit), and check their docstring.

Reply:
> Do you want to change all the docsting inside builder app?

Oh, I think I made some mistakes when implementing the builder before.
Please fix them😅

Reply:
> I will add dummy 4 scan type classes (Range, Center, NoScan, Explicit), and check their docstring.

If you plan this in next PR, please leave a TODO comment.

Reply:
Also, if `_ScanEntry` is a dummy class and it will be removed later, I think the detailed `argInfo` description is unnecessary.